src/components/basic/SelectList/index.tsx

```diff
@@ -60,6 +61,7 @@ export const SelectList = ({
   onChangeItem,
 }: SelectListProps) => {
   const selectHeight = popperHeight ? `${popperHeight}px` : 'auto'
+  const [selected, setSelected] = useState<any>({})
```

Comment:
please stop using `any`- if necessary create a proper type and export it. If there was `any` before then we take the time to clean up dirty coding.

Reply:
this selected can only be `any`. We are not sure about the `items` type in this component.